drakcore/drakcore/postprocess/apicall.py

```diff
@@ -0,0 +1,31 @@
+import json
+from io import BytesIO
+from collections import defaultdict
+from drakcore.postprocess import postprocess
+from karton2 import Task, Resource
+from typing import Dict
+
+
+def process_logfile(log):
+    result = []
+
+    for line in log.readlines():
+        entry = json.loads(line)
+        r = {
+            "pid": entry["PID"],
+            "timestamp": entry["TimeStamp"],
+            "method": entry["Method"],
+            "arguments": entry["Arguments"],
+        }
+        result.append(r)
+    return result
+
+
+@postprocess(required=["apimon.log"])
+def process_api_log(task: Task, resources: Dict[str, Resource], minio):
+    res_log = resources["apimon.log"]
+    log = BytesIO(res_log.content)
+    data = json.dumps(process_logfile(log)).encode()
+    file = BytesIO(data)
+    analysis_uid = task.payload["analysis_uid"]
+    minio.put_object("drakrun", f"{analysis_uid}/apicall.json", file, len(data))
```

Comment:
This causes copying on the whole log multiple times, I would rather stick with [ndjson](http://ndjson.org/) format because then, the file could be processed on a line-by-line basis. With a regular JSON it's necessary to load everything to RAM which is not the best idea when the log is very heavy.